### PR TITLE
ci: Fix CI for forks

### PR DESCRIPTION
**Pull request by Twixes**
## Problem

A few CI checks are failing on PRs from forks (e.g. see https://github.com/PostHog/posthog/pull/16542).

## Changes

Fixing this.

### CODE REVIEW

**Comment by nategrift** [comment]
Love this and very helpful, thank you!

File: .github/actions/run-backend-tests/action.yml [hunk 1]
```diff
@@ -2,29 +2,28 @@
 # This is a composite action that packages our backend Django tests.
 # It is called by the `ci-backend.yml` job using a matrix.
 #
-name: Run Backend Django tests
+name: Run Django tests
 inputs:
     python-version:
         required: true
-        type: string
```
Comment (Twixes) [inline .github/actions/run-backend-tests/action.yml:9]: Action inputs don't actually have a `type` parameter (they're all strings for GitHub)

**Review by pauldambra — APPROVED** [review]
You're so good at this tricky CI fangling 💖 

I guess the real test is in a fork once this is in... :shipit:

**Comment by Twixes** [comment]
This _is_ a fork @pauldambra ;) Otherwise I'd have no idea if this makes sense

**Comment by pauldambra** [comment]
OH MY GOD 🥇

**Comment by neilkakkar** [comment]
@nategrift once this is in, can you merge master into your PRs so we can merge? :)

**Pull request merged**